<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1073</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="PrincipalCapteurGL" name="glPrincipal" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>531</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>531</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QSlider" name="horizontalSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>430</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>430</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>430</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="CapteurGL" name="glSignalAcc" native="true">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>150</y>
      <width>511</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>230</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Accéléromètre : Evolution de l'accélération</string>
    </property>
   </widget>
   <widget class="CapteurGL" name="glSignalGyro" native="true">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>270</y>
      <width>511</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>350</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Gyroscope : Evolution de la vitesse angulaire</string>
    </property>
   </widget>
   <widget class="CapteurGL" name="glSignalMagne" native="true">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>380</y>
      <width>511</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>460</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Magnétomètre : évolution</string>
    </property>
   </widget>
   <widget class="AccCapteurGL" name="glCapteurAcc" native="true">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>60</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="GyroCapteurGL" name="glCapteurGyro" native="true">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>60</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="MagneCapteurGL" name="glCapteurMagne" native="true">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>60</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>204</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="1">
      <widget class="QComboBox" name="pOsitionDuCapteurComboBox">
       <item>
        <property name="text">
         <string>Pied</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Poche</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Main</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="pOsitionDuCapteurLabel">
       <property name="text">
        <string>Position du capteur</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="CapteurGL" name="widget_8" native="true">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>10</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>20</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Type du mouvement détécté</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1073</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuIMUViewer">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionCharger_un_nouveau_fichier"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menu_propos">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionAide"/>
    <addaction name="actionA_propos"/>
    <addaction name="actionCredits"/>
   </widget>
   <addaction name="menuIMUViewer"/>
   <addaction name="menu_propos"/>
  </widget>
  <action name="actionCharger_un_nouveau_fichier">
   <property name="text">
    <string>Charger un nouveau fichier</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionAide">
   <property name="text">
    <string>Aide</string>
   </property>
  </action>
  <action name="actionA_propos">
   <property name="text">
    <string>A propos</string>
   </property>
  </action>
  <action name="actionCredits">
   <property name="text">
    <string>Credits</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CapteurGL</class>
   <extends>QWidget</extends>
   <header>capteurgl.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
